test/jdk/jdk/jfr/event/profiling/TestFullStackTrace.java

```diff
@@ -41,13 +41,15 @@
  * @key jfr
  * @requires vm.hasJFR
  * @library /test/lib
- * @run main/othervm jdk.jfr.event.profiling.TestFullStackTrace
+ * @run main/othervm jdk.jfr.event.profiling.TestFullStackTrace wall-clock
```

Comment:
Create separate tests for CPU time sampling events. It makes them easier to problem list and disable for specific platforms. If you want to share verification code, create  helper class.

Reply:
I created two test classes and moved almost all code to `BaseTestFullStackTrace`.